checker/checker.ml

```diff
@@ -181,7 +181,7 @@ let print_usage_channel co command =
 \n  -Q dir coqdir               map physical dir to logical coqdir\
 \n  -R dir coqdir               synonymous for -Q\
 \n  -coqlib dir                 set coqchk's standard library location\
-\n  -boot                       don't initalize the library paths automatically\
+\n  -boot                       don't initialize the library paths automatically\
```

Comment:
Unrelated to PR title

Reply:
Picky, picky.  Changed it.

Reply:
@SkySkimmer Can you be the assignee?  Or suggest someone?  Thanks.